src/Middleware/HttpLogging/src/FileLoggerProcessor.cs

```diff
@@ -309,13 +337,13 @@ private static class Log
 
             public static void CreateDirectoryFailed(ILogger logger, string path, Exception ex) => _createDirectoryFailed(logger, path, ex);
 
-            private static readonly Action<ILogger, Exception> _maxFilesReached =
+            private static readonly Action<ILogger, Exception?> _maxFilesReached =
                 LoggerMessage.Define(
                     LogLevel.Warning,
                     new EventId(3, "MaxFilesReached"),
-                    "Limit of 10,000 files per day has been reached");
+                    $"Limit of {W3CLoggerOptions.MaxFileCount} files per day has been reached");
```

Comment:
We almost never use string interpolation in these pre-defined log messages unless the value we're writing here will never change. In this case the options can be updated so we don't want to embed the original value in the log.

We should pass the option to `MaxFilesReached` and also remove the `Exception` param, I don't think it's needed anywhere?

Reply:
`MaxFileCount` can't be set by the user, it's hard-coded to 10,000. It's the max number of files per day before we stop logging - different from `RetainedFileCountLimit`, which determines when we start rolling files & can't be greater than `MaxFileCount`.

Reply:
I removed the exception but kept the line you're commenting on

Reply:
> `MaxFileCount` can't be set by the user

Ah, thought that was a settable option, log changes look good now 👍